recipe_scrapers/donnahay.py

```diff
@@ -30,7 +30,14 @@ def ingredients(self):
         return self.schema.ingredients()
 
     def instructions(self):
-        return self.schema.instructions()
+        div = self.soup.find("div", class_="col-sm-6 method")
+        if not div:
+            return
+        instructions = div.find_all("li")
+        for instruction in instructions:
+            [tag.extract() for tag in instruction.find_all("b") if "Serves" in tag.get_text()] # Removes any bold text with 'Serves' in it
```

Comment:
Any ideas what could be done when the serve amount is represented like this?
https://www.donnahay.com.au/recipes/dinner/fish-and-seafood/baked-mussels-with-tomato-and-capers

Reply:
I will have a look into it, maybe removing all sentences that have 'Serves' followed by a digit?